### PR TITLE
Adding source code for article tracked under BAEL-4109.

### PR DESCRIPTION
**Pull request by kcacademic**

### CODE REVIEW

File: spring-webflux-threads/src/main/java/com/baeldung/webflux/Controller.java [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+
+    @GetMapping("/threads/mongodb")
+    public Flux<String> getIndexMongo() {
+        personRepository.findAll()
+            .doOnNext(p -> logger.info("Person: {}", p))
+            .subscribe();
+        return Flux.fromIterable(getThreads());
+    }
+
+    @GetMapping("/thareds/reactor-kafka")
```
Comment (bfontana) [inline spring-webflux-threads/src/main/java/com/baeldung/webflux/Controller.java:84]: thareds -> theads

File: spring-webflux-threads/pom.xml [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+        <artifactId>parent-boot-2</artifactId>
+        <version>0.0.1-SNAPSHOT</version>
+        <relativePath>../parent-boot-2</relativePath>
+    </parent>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-webflux</artifactId>
+            <!--
```
Comment (bfontana) [inline spring-webflux-threads/pom.xml:23]: Delete commented out lines. The same happens below

File: spring-webflux-threads/pom.xml [hunk 3]
```diff
@@ -0,0 +1,87 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>com.baeldung.spring</groupId>
+    <artifactId>spring-webflux-threads</artifactId>
+    <version>1.0.0-SNAPSHOT</version>
+    <name>spring-webflux-threads</name>
+    <packaging>jar</packaging>
+    <description>Spring WebFlux AMQP Sample</description>
```
Comment (bfontana) [inline spring-webflux-threads/pom.xml:10]: Spring WebFlux Threads Sample

File: spring-webflux-threads/src/main/resources/application.yml [hunk 4]
```diff
@@ -0,0 +1,7 @@
+
```
Comment (bfontana) [inline spring-webflux-threads/src/main/resources/application.yml:1]: Can we delete this empty file or it's necessary?

Comment (kcacademic) [reply spring-webflux-threads/src/main/resources/application.yml:1]: Yes, we can, this is a default boot properties class.

File: spring-webflux-threads/pom.xml (hunk 3 above)
Comment (kcacademic) [reply spring-webflux-threads/pom.xml:10]: Corrected

File: spring-webflux-threads/pom.xml (hunk 2 above)
Comment (kcacademic) [reply spring-webflux-threads/pom.xml:23]: The commented lines are actually a way to suggest how to change the default runtime.

File: spring-webflux-threads/src/main/java/com/baeldung/webflux/Controller.java (hunk 1 above)
Comment (kcacademic) [reply spring-webflux-threads/src/main/java/com/baeldung/webflux/Controller.java:84]: Corrected

File: spring-webflux-threads/pom.xml (hunk 2 above)
Comment (kcacademic) [reply spring-webflux-threads/pom.xml:23]: I have added comments in the pom to indicate why the lines are commented.

**Pull request merged**